### PR TITLE
Don't warn about missing WCSs when none are expected

### PR DESCRIPTION
**Pull request by RobertLuptonTheGood**

### CODE REVIEW

File: python/lsst/ip/isr/isrTask.py [hunk 1]
```diff
@@ -188,6 +188,11 @@ class IsrTaskConfig(pexConfig.Config):
         default=True,
         doc="Assemble amp-level exposures into a ccd-level exposure?"
     )
+    expectWcs = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Input science images are expected to have a WCS (not true for e.g. spectrographs)"
```
Comment (parejkoj) [inline python/lsst/ip/isr/isrTask.py]: I'd phrase this "Expect input science images to have a WCS (set False for e.g. spectrographs)".

File: python/lsst/ip/isr/isrTask.py [hunk 2]
```diff
@@ -426,6 +431,8 @@ def run(self, ccdExposure, bias=None, linearizer=None, dark=None, flat=None, def
 
         if self.config.doAssembleCcd:
             ccdExposure = self.assembleCcd.assembleCcd(ccdExposure)
+            if not ccdExposure.getWcs() and self.config.expectWcs:
```
Comment (parejkoj) [inline python/lsst/ip/isr/isrTask.py]: It doesn't really matter, but I'd reverse the order here to short-circuit on getting the WCS if it's not necessary.

File: python/lsst/ip/isr/isrTask.py [hunk 3]
```diff
@@ -426,6 +431,8 @@ def run(self, ccdExposure, bias=None, linearizer=None, dark=None, flat=None, def
...
+                self.log.warn("No WCS found in input exposure")
```
Comment (parejkoj) [inline python/lsst/ip/isr/isrTask.py]: Should we include in the warning which exposure had the problem?

Comment (RobertLuptonTheGood) [inline python/lsst/ip/isr/isrTask.py]: There isn't enough context, but if you run the task with `--longlog` you'll get what you need.

**Pull request merged**